utils/elasticsearch.js

```diff
@@ -0,0 +1,19 @@
+const { Client } = require('@elastic/elasticsearch');
+const { Bash } = require('./bash');
+
+const service = 'aws-es-proxy-deployment';
+const bash = new Bash();
+const ES_URL = 'http://localhost:9200';
+
+module.exports = {
+  search: async (index, query) => {
+    const res = bash.runCommand(`g3kubectl port-forward deployment/${service} 9200:9200 &`);
```

Comment:
Maybe get a +1 from DevOps to gather caveats on potential accumulation of kubectl port-forward processes in the background? 🤷‍♂

Reply:
it should only run once - you should get an error if you retry - Error listen tcp6 [::1]:9200: bind: address already in use]

Reply:
Concurrent PR checks + potential executions from local terminals can create multiple instances of the same command, right?

Reply:
The command will fail to run a second time.